composer.json

```diff
@@ -418,9 +418,6 @@
                 "CKeditor Plugin - using a combined patch to avoid conflict (https://www.drupal.org/node/2928318)": "https://www.drupal.org/files/issues/2020-10-28/media_entity_download-ckeditor_plugin-2928318-20.patch",
                 "3106062 - Track the usage of Media entity through media_entity_download route with entity_usage": "patches/media_entity_download-track_media_entity_usage-3106062-2.patch"
             },
-            "drupal/maxlength": {
-                "Don't enforce maxlength stop, allow negative count down": "https://www.drupal.org/files/issues/2020-12-07/2916687-allow-negative-count-13.patch"
```

Comment:
Why we didn't replace this patch with the latest one compatible with the 2.x version of the module, we might break something by just removing the patch.

Reply:
After some investigation I think that we can actually remove this patch, since there is an implementation already in 2.x branch, however that requires additional changes from the UI.